release-notes/major9/09.20.0/valtimo-backend-libraries.md

```diff
@@ -11,7 +11,13 @@ The following features were added:
   `DocumentSearchService`, this method has a default and only needs to be implemented if this use case is necessary.
 
 * **Zaakobjecten can now be edited**
+
   When creating a form that includes a propertyName with "zaakobject:objecttype:path/to/variable" it will now be
   prefilled when loading the form. When submitting the form the corresponding zaakobject is updated with the new values
   
   The uploading is handled by a new `ExternalDataSubmittedEvent` handler called `ZaakObjectListener`
+
+Changes:
```

Comment:
Don't think this is necessary. Can you remove it?

Reply:
I put it there because at the top on line 5 there's: "The following features were added"

We didn't add a feature. So either both lines go, it it stays imho.

Reply:
removed it.